client/pokemon/pokemon.ctrl.js

```diff
@@ -95,25 +95,29 @@ module.exports = function($routeParams, $scope, io) {
         moveName: this.data.move1Name,
         moveType: this.data.move1Type,
         Pp: this.data.move1Pp,
-        Ppu: this.data.move1Ppu
+        Ppu: this.data.move1Ppu,
+        power: this.data.move1Power
       },
       {
         moveName: this.data.move2Name,
         moveType: this.data.move2Type,
         Pp: this.data.move2Pp,
-        Ppu: this.data.move2Ppu
+        Ppu: this.data.move2Ppu,
+        power: this.data.move1Power
```

Comment:
make sure to update these to `move1Power`, `move2Power`, `move3Power`, and `move4Power`. At the moment they all say `move1Power`.